### PR TITLE
add sgml-base and xml-core

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: recipes-debian/sources/discover.inc [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# This is generated by debian-source.bbclass
```
Comment (hiraku-wfs) [inline recipes-debian/sources/discover.inc]: Is this file unrelated to the PR?

Comment (yuwata) [reply recipes-debian/sources/discover.inc]: Ouch.

Comment (yuwata) [reply recipes-debian/sources/discover.inc]: Dropped.

**Comment by hiraku-wfs** [comment]
LGTM,

**Pull request merged**